model/event/event.go

```diff
@@ -122,8 +122,8 @@ func (e *EventLogEntry) validateEvent() error {
 	if len(e.ResourceType) == 0 {
 		return errors.New("event log entry has no r_type")
 	}
-	if !e.ID.Valid() {
-		e.ID = bson.NewObjectId()
+	if e.ID == "" {
+		e.ID = bson.NewObjectId().String()
```

Comment:
I thought the whole point was to remove the dependency on bson? Additionally, please add tests to ensure the unmarshaller can successfully unmarshal existing events with bson.ObjectId's

Reply:
That's the goal eventually, but the scope of this ticket is to have the IDs not be ObjectIDs. The existing tests cover unmarshalling of ObjectIDs already - for example TestWithRealData